### DISCUSSION TITLE
Seizures and Sleep Patterns

### DISCUSSION THREAD

**Discussion by jones139** [OpenSeizureDetector/Android_Pebble_SD#71] [06_Epilepsy_Research]
Eduardo has pointed out that the paper below concludes: “_REM sleep has been found in particular to play a seizure-protective role, and therefore seizures during this phase are a rare phenomenon. Multiple pathways have been postulated that makes REM sleep relatively immune to seizures, when compared to other stages of sleep as well as the awake state._”

https://n.neurology.org/content/90/15_Supplement/P3.278

Maybe there is some benefit in using the sleep tracking functions of some wearables in seizure detection?

**Comment by AroonPro** [comment]
Using Google libraries, it should be possible to cross-over. Latest tool is already available to cross-over from Samsung Health to Google-Fit. Android Studio has also Google-Fit libraries available.

NL-AVG EU-GDPA rules apply with handling data.
